src/kinetics/RxnRates.cpp

```diff
@@ -68,6 +69,23 @@ void Arrhenius::getParameters(AnyMap& rateNode, const Units& rate_units) const
     rateNode.setFlowStyle();
 }
 
+void Arrhenius::validate(const std::string& equation) {
+    fmt::memory_buffer err_reactions;
+    double T[] = {200.0, 500.0, 1000.0, 2000.0, 5000.0, 10000.0};
+    for (size_t i=0; i < 6; i++) {
+        double gamma = m_A * std::exp(m_b*log(T[i]) - m_E*(1/T[i]));
+        if (gamma > 1) {
+             format_to(err_reactions,
+                       "\n Sticking coefficient is more than 1 for reaction '{}'\n"
+                       " at T = {:.1f}\n",
+                       equation, T[i]);
+        }
+    }
+    if (err_reactions.size()) {
+        throw CanteraError("Arrhenius::validate", to_string(err_reactions));
```

Comment:
With the introduction of YAML, `InputFileError` provides a mechanism to point directly to the YAML section where the issue occurs (although the exception still works for CTI).